app/models/product_import/entry_validator.rb

```diff
@@ -298,6 +299,11 @@ def attributes_match?(attribute, existing_product, entry)
       existing_product_value == convert_to_trusted_type(entry_value, existing_product_value)
     end
 
+    def ignore_when_updating_product?(attribute)
+      attributes_to_ignore = [:description]
```

Comment:
I know this is a micro-optimization but these things are better suited as a constant so they are evaluated only once and not every single time the method is called. Something like 

```ruby
class EntryValidator
  ATTRIBUTES_TO_IGNORE = [:description].freeze
  (...)
  def ignore_when_updating_product?(attribute)
    ATTRIBUTES_TO_IGNORE.include? attribute
  end
end
```